examples/src/main/java/org/apache/ignite/examples/ml/knn/KNNClassificationExample.java

```diff
@@ -60,7 +60,8 @@ public static void main(String[] args) throws IOException {
                 KNNClassificationTrainer trainer = new KNNClassificationTrainer()
                     .withK(3)
                     .withDistanceMeasure(new EuclideanDistance())
-                    .withWeighted(true);
+                    .withWeighted(true)
+                    .withDataTtl(60);
```

Comment:
I suggest to move it to DataSet builder or to DataConfiguration (it seems strange in Trainer)

Reply:
By default, it's part of the learning environment. In the case of kNN it's also been added into trainer API because it's critical for this algorithm.